cmd/segment-reaper/delete_test.go

```diff
@@ -0,0 +1,81 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package main
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/gogo/protobuf/proto"
+	"github.com/stretchr/testify/require"
+
+	"storj.io/storj/pkg/pb"
+	"storj.io/storj/private/testcontext"
+	"storj.io/storj/satellite/metainfo"
+	"storj.io/storj/storage"
+	"storj.io/storj/storage/teststore"
+)
+
+func TestDeleteSegment(t *testing.T) {
+	ctx := testcontext.New(t)
+	defer ctx.Cleanup()
+
+	db := teststore.New()
+	defer ctx.Check(db.Close)
+
+	{
```

Comment:
nit: I would create a subtest for each of these scopes and use the comments that you have in the last `require` assertion of each scope as the subtest name.

Reply:
I reorganized tests, let me know what do you think

Reply:
They look good!